cmd/root.go

```diff
@@ -24,6 +24,8 @@ import (
 
 	"github.com/ethereum/go-ethereum/ethclient"
 	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/evalphobia/logrus_sentry"
```

Comment:
Found this at https://github.com/sirupsen/logrus/wiki/Hooks but I'd be fine with hand rolling our own if we don't want to depend on a repo we don't own for this behavior